Integrations/integration-CrowdStrikeFalconSandbox.yml

```diff
@@ -772,6 +772,9 @@ script:
         case 'vx-submit-sample': // Deprecated
         case 'crowdstrike-submit-sample':
             var file = submitFile(args.entryId, args.environmentID);
+            if ('Type' in file && file['Type'] === entryTypes.error) {
```

Comment:
please find better name then file, because this is clearly not a file

Reply:
done